packages/main/src/Menu.ts

```diff
@@ -542,9 +546,16 @@ class Menu extends UI5Element {
 		this._parentItemsStack.push(item);
 	}
 
+	_onfocusin(e: CustomEvent<any>): void {
+		const target = e.target as HTMLElement;
+		const menuListItem = target instanceof MenuListItem ? target : (target.getRootNode() as any).host;
+		const menuItem = menuListItem.associatedItem as MenuItem;
+		const mainMenu = this._findMainMenu(menuItem);
+		mainMenu?.fireEvent("ui5-item-focusin", { opener: menuListItem, menuItem });
+	}
+
 	_startOpenTimeout(item: MenuItem, opener: OpenerStandardListItem, hoverId: string) {
-		// If theres already a timeout, clears it
-		this._clearTimeout();
+		clearTimeout(this._timeout);
```

Comment:
Why was this done? The `_clearTimeout` function is the same but with a runtime check.

Reply:
The runtime check is redundant as `clearTimeout` native method knowns by design how to act with an undefined argument.

Reply:
Makes sense, then let's delete this function